backend/src/jobs_server/services/k8s.py

```diff
@@ -53,12 +53,15 @@ def workload_for_managed_resource(
         except WorkloadNotFound:
             return None
 
+    def _sanitize_log_kwargs(self, tail: int) -> dict[str, Any | None]:
```

Comment:
This would be a `dict[str, int]` as far as I am concerned (the empty dict in the `else` branch below is a trivial instance of that type).

Reply:
True, I let it open on purpose in case there are other parameters we want to sanitize. 

Though we can also adjust the type once we get to that bridge. 

Agree w.r.t to the none type.